From 0d425bc96fb08f0e5c61527796894909bd24da47 Mon Sep 17 00:00:00 2001
From: Joachim Breuer <git@jmbreuer.net>
Date: Mon, 22 May 2023 17:45:55 +0200
Subject: [PATCH] work around compile error

gcc (Gentoo 12.2.1_p20230428-r1 p2) balks at some uses of
FMT_STRING_IMPL in src/third_party/fmt/format.h
used by LOG() and FMT() macros
---
 src/storage/local/LocalStorage.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/storage/local/LocalStorage.cpp b/src/storage/local/LocalStorage.cpp
index fe423989fe..d106f04023 100644
--- a/src/storage/local/LocalStorage.cpp
+++ b/src/storage/local/LocalStorage.cpp
@@ -707,7 +707,7 @@ LocalStorage::recompress(const std::optional<int8_t> level,
           auto l2_content_lock = get_level_2_content_lock(l1_index, l2_index);
           l2_content_lock.make_long_lived(lock_manager);
           if (!l2_content_lock.acquire()) {
-            LOG("Failed to acquire content lock for {}/{}", l1_index, l2_index);
+            LOG_RAW(fmt::format("Failed to acquire content lock for {}/{}", l1_index, l2_index));
             return;
           }
 
